### PR TITLE
Just a small PR to get startet

### PR DESCRIPTION
**Pull request by kmthorsnes**
* Adds JS in footer for updating year
* Use .svg instead of .png for logo.
* Small adjustments to readme.md
* Adds some colors that is the [netflix color palette](https://www.color-hex.com/color-palette/22942)
* I tried to add the colors into tailwind classes as described [here](https://tailwindcss.com/docs/customizing-colors), but could not get it to work

* I see that my code formatter (prettier) changed to standard of the index.html, sorry. Do you use a code formatter yourself?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,30 +1,42 @@
...
+        </ul>
+      </nav>
+    </header>
+    <main class="p-20">
+      <p class="util-white">Movie listings go here...</p>
+    </main>
+    <footer class="bg-gray-700 text-center p-10 ">
+      <p>
+        &COPY; Copyright
+        <script type="text/javascript">
```
Comment (adamszs) [inline index.html:34]: Going to update this text into the main.js file, I dont like using inline scripting unless its a tracker or above the fold for css.

File: style.css [hunk 2]
```diff
@@ -1,6 +1,8 @@
-.util-primary { color: #E50914;}
-.util-red { color: #b91d24;}
-.util-black {color: #221f1f;}
-.util-white {color: #f5f5f1;}
-
-nav ul li a {margin:auto 15px;}
+.util-primary { color: #db0000;}
```
Comment (adamszs) [inline style.css:1]: where are you getting your color codes from? I got them directly from Netflix Branding Guide.

**Review by adamszs — APPROVED** [review]
going to approve but add another pull request for updating main.js file.

File: index.html (hunk 1 above)
Comment (kmthorsnes) [reply index.html:34]: Agreed.

File: style.css (hunk 2 above)
Comment (kmthorsnes) [reply style.css:1]: Took it from here: [https://www.color-hex.com/color-palette/22942](https://www.color-hex.com/color-palette/22942). That being said, your source being the official documentation is of course better than mine

**Comment by kmthorsnes** [comment]
> going to approve but add another pull request for updating main.js file.

Seems like you have taken care of it in the main branch already(?).

**Pull request merged**